### PR TITLE
inital version of app

### PR DESCRIPTION
**Pull request by chandrahmani**
Have sliders

responsive supports

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+
+<body>
+  <div id="root"></div>
+  <script type="module" src="/src/main.jsx"></script>
+</body>
+
+
+
+
```
Comment (hidaytrahman) [inline index.html:20]: remove extra space

File: index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+  <title>Vite + React</title>
```
Comment (hidaytrahman) [inline index.html:10]: Change name to Hello Travel India

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-axios": "^2.0.6",
```
Comment (hidaytrahman) [inline package.json:13]: use `axios` instead of `react-axios`

File: src/components/header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
+import { useForm } from "react-hook-form"
+import TravelDtl from "../travelDtl/TravelDtl";
+import "./Header.css"
+const Header = () => {
+
+    const { register, handleSubmit, formState: { errors } } = useForm();
+
+    const onSubmit = () => {
+
```
Comment (hidaytrahman) [inline src/components/header/Header.jsx:9]: Why this is blank

File: src/components/service/Service.css [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+.service-section {
+    padding: 50px 0;
+    position: relative;
+    background: #f0f0f0;
+
+}
+
+.heading-dtl h1 {
+    letter-spacing: 0px;
```
Comment (hidaytrahman) [inline src/components/service/Service.css:38]: always remove px if its 0

File: src/components/travelDtl/TravelDtl.jsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react"
+import Servic from "../service/Servic"
+import './Traval.css'
+import axios from "axios";
+
+
+// const pth = "../../data/index.json"
```
Comment (hidaytrahman) [inline src/components/travelDtl/TravelDtl.jsx:7]: 🤖

File: src/components/header/Header.jsx (hunk 4 above)
Comment (chandrahmani) [reply src/components/header/Header.jsx:9]: I will utilize it later

**Comment by chandrahmani** [comment]
@hidaytrahman can you please approve this i will do in the separate PR

**Review by hidaytrahman — APPROVED** [review]
yes please do all the suggested changes in the next PR

**Comment by hidaytrahman** [comment]
## About Git

image

**Pull request merged**